packages/management/tkg-clusterclass/bundle/config/packageinstalls/service-account.yaml

```diff
@@ -0,0 +1,75 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tanzu-infra-clusterclass-package-sa
+  namespace: tkg-system
+  annotations:
+    kapp.k14s.io/change-group: "v1/serviceaccount"
```

Comment:
just curious, is it okay to use the same ordering annotation in different packages? not sure if it affects resource creation/deletion

Reply:
Thanks for the suggestion. I think it would be better to keep different annotation to differentiate things a bit more.

Reply:
Updated PR